RESTSystem/API/CS/CSHandler.py

```diff
@@ -0,0 +1,35 @@
+import types
+import os
+import shutil
+import json
+from tornado import web, gen
+from RESTDIRAC.RESTSystem.Base.RESTHandler import WErr, WOK, TmpDir, RESTHandler
+from RESTDIRAC.ConfigurationSystem.Client.Helpers import RESTConf
+
+class CSHandler( RESTHandler ):
+
+  ROUTE = "/config/(Sections|Options|Value)"
+
+  @web.asynchronous
+  def get( self, reqType ):
+    if reqType == "Sections":
+      return self.SectionsAction()
+    elif reqType == "Options":
+      return self.OptionsAction()
+    elif reqType == "Value":
+      return self.ValueAction()
```

Comment:
I marked this as a "WIP" (Work In progress) PR, when finished please remove the tag